libr/core/panels.c

```diff
@@ -363,6 +367,7 @@ static bool __check_if_mouse_x_on_edge(RCore *core, int x, int y) {
 	int i = 0;
 	for (; i < panels->n_panels; i++) {
 		RPanel *panel = __get_panel (panels, i);
+		if (!panel) continue;
```

Comment:
please follow the coding style guide. there are no single line ifs. always add braces and a newline